staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/util/client_config.go

```diff
@@ -33,6 +33,7 @@ func HookClientConfigForWebhook(w *v1beta1.Webhook) webhook.ClientConfig {
 		ret.Service = &webhook.ClientConfigService{
 			Name:      w.ClientConfig.Service.Name,
 			Namespace: w.ClientConfig.Service.Namespace,
+			Port:      *w.ClientConfig.Service.Port,
```

Comment:
an aggregated API server running against an n-1 kube-apiserver would panic here, right?

Reply:
v1beta1.Webhook is an admissionregistration object that should have been stored or accepted by the user. In both cases it will be defaulted before it gets here. how does it panic here?